runtime/src/accounts_db.rs

```diff
@@ -169,7 +169,7 @@ struct FoundStoredAccount<'a> {
 #[cfg(not(test))]
 const ABSURD_CONSECUTIVE_FAILED_ITERATIONS: usize = 100;
 
-type DashMapVersionHash = DashMap<Pubkey, (u64, Hash)>;
+type DashMapVersionHash = DashMap<Pubkey, (u64, Hash, bool)>;
```

Comment:
The value type would probably be better off as a struct with named fields

Reply:
this has been totally reworked